tasks/18_humanize-format/humanize-format.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * humanizeFormat - returns number with the correct
+ * suffix such as 1st, 2nd, 3rd or 4th
+ *
+ * @param {Number} number
+ * @return {String}
+ */
+export function humanizeFormat(number) {
+  if (number === undefined) {
+    return '';
+  }
+  let stringNumber = number.toString();
+  let lastChar = stringNumber.substring(
+      stringNumber.length - 1, stringNumber.length);
+  if (lastChar === '1') {
```

Comment:
I think you could use `switch` in this case.

Reply:
Fixed